version/chrome/manifest.json

```
@@ -1,66 +1,8 @@
 {
-    "name": "NopeCHA: CAPTCHA Solver",
-    "action": {
-        "default_title": "NopeCHA: CAPTCHA Solver",
-        "default_icon": "icon/16.png",
-        "default_popup": "popup.html"
-    },
     "manifest_version": 3,
-    "version": "0.1.18",
-    "description": "Automatically solve reCAPTCHA and hCaptcha using AI.",
     "permissions": [
-        "declarativeNetRequest",
-        "storage",
-        "scripting"
+        "declarativeNetRequest"
     ],
-    "background": {"service_worker": "background.js"},
-    "content_scripts": [
-        {
-            "matches": ["*://*.hcaptcha.com/captcha/*"],
-            "js": ["hcaptcha.js"],
-            "all_frames": true,
-            "run_at": "document_end"
-        },
-        {
-            "matches": ["*://*.hcaptcha.com/captcha/*"],
-            "js": ["hcaptcha_fast.js"],
-            "all_frames": true,
-            "run_at": "document_start"
-        },
-        {
-            "matches": ["<all_urls>"],
-            "js": ["recaptcha.js", "recaptcha_voice.js"],
-            "all_frames": true,
-            "run_at": "document_end"
-        },
-        {
-            "matches": ["<all_urls>"],
-            "js": ["hcaptcha_language.js"],
-            "all_frames": true,
-            "run_at": "document_end"
-        },
-        {
-            "matches": [
-                "*://*.google.com/recaptcha/*",
-                "*://*.recaptcha.net/recaptcha/*",
-                "*://recaptcha.net/recaptcha/*"
-            ],
-            "js": ["recaptcha_fast.js"],
-            "all_frames": true,
-            "run_at": "document_start"
-        },
-        {
-            "matches": ["*://nopecha.com/setup"],
-            "js": ["setup.js"],
-            "all_frames": true,
-            "run_at": "document_end"
-        }
-    ],
-    "host_permissions": ["<all_urls>"],
-    "icons": {
-        "16": "icon/16.png",
-        "32": "icon/32.png",
-        "48": "icon/48.png",
-        "128": "icon/128.png"
-    }
+    "background": {"service_worker": "version/chrome/background.js"},
```

```suggestion
    "background": {"service_worker": "background.js"},
```
this should be relative to the manifest

---

Unless you are changing the location of the `background.js` and I missed it, the path I wrote is correct. There's no `background.js` at the root of the directory.